packages/web/lib/browser.js

```diff
@@ -8,20 +10,49 @@ const {
   resolveFn,
   resolveFnScope,
   resolveInternalScope,
+  validateScope,
 } = require("@browselang/core/lib/scope");
 const { help } = require("@browselang/core/lib/utils");
 const UrlPattern = require("url-pattern");
 const { BrowseError } = require("@browselang/core/lib/error");
 
 const getPageScope = require("./page");
 
+//Centralize control of goto function
```

Comment:
"A more helpful page.goto" seems like a better comment. Just a nitpick. upto you